app/_hub/kong-inc/http-log/index.md

```diff
@@ -66,33 +60,58 @@ params:
       required: true
       default:
       value_in_examples: http://mockbin.org/bin/:id
-      description: The HTTP endpoint (including the protocol to use) to which the data will be sent.
+      description: The HTTP endpoint (including the protocol to use) to which the data is sent.
     - name: method
       required: false
       default: "`POST`"
       value_in_examples: POST
       description: |
-        An optional method used to send data to the http server, other supported values are `PUT`, `PATCH`
+        An optional method used to send data to the HTTP server. Supported values are `POST` (default), `PUT`, and `PATCH`.
+    - name: content_type
+      required: true
+      default: "application/json"
+      value_in_examples: application/json
+      description: |
+        REVIEWERS: need a description.          
     - name: timeout
       required: false
       default: "`10000`"
       value_in_examples: 1000
-      description: An optional timeout in milliseconds when sending data to the upstream server
+      description: An optional timeout in milliseconds when sending data to the upstream server.
     - name: keepalive
       required: false
       default: "`60000`"
       value_in_examples: 1000
-      description: An optional value in milliseconds that defines for how long an idle connection will live before being closed
+      description: An optional value in milliseconds that defines how long an idle connection will live before being closed.
+    - name: flush_timeout
+      required:
+      default: "`2`"
+      value_in_examples: 2
+      description: |  
+        REVIEWERS: need a description.      
+    - name: retry_count
+      required: false
+      default: 10
+      value_in_examples: 15
+      description: Number of times to retry when sending data to the upstream server.
+    - name: queue_size
+      required: false
+      default: 1
+      description: Max number of log entries to be sent on each message to the upstream sever.
+    - name: headers
+      required: false
+      default: empty table
+      description: An optional table of headers added to the HTTP message to the upstream server.
   extra: |
-    **NOTE:** If the `config.http_endpoint` contains a username and password (ex.
-    `http://bob:password@example.com/logs`), then Kong will automatically include
+    **NOTE:** If the `config.http_endpoint` contains a username and password (for example,
+    `http://bob:password@example.com/logs`), then Kong Gateway automatically includes
     a basic-auth `Authorization` header in the log requests.
 
 ---
 
 ## Log Format
 
-Every request will be logged separately in a JSON object, with the following format:
+Every request is logged separately in a JSON object in the following format:
```

Comment:
> if the `queue_size` argument is >1, the payload will be a JSON array of objects in this format. even if there's only one record (because flush_timeout has expired before getting a second request), it will still be a one-element array.

Adjusted in commit 0c00b76. Please adjust as you see fit @javierguerragiraldez. Thank you kindly for collaborating on these updates!